modular_zubbers/code/modules/cargo/packs/security.dm

```diff
@@ -25,3 +25,61 @@
 	cost = CARGO_CRATE_VALUE * 7
 	contains = list(/obj/item/ammo_box/magazine/wt550m9 = 3)
 	crate_name = "wt-550 magazine crate (regular)"
+
+/datum/supply_pack/security/armory/renoster
+	name = "Carwo 'Renoster' Shotgun Crate"
+	desc = "Contains two Carwo 'Renoster' shotguns. Additional ammmo sold separately."
+	cost = CARGO_CRATE_VALUE * 20
+	contains = list(/obj/item/gun/ballistic/shotgun/riot/sol = 2,
+	/obj/item/ammo_box/advanced/s12gauge/buckshot = 2,
+	)
+	crate_name = "Carwo 'Renoster' Shotgun Crate"
+
+/datum/supply_pack/security/armory/sindano
+	name = "Carwo 'Sindano' Submachinegun Crate"
+	cost = CARGO_CRATE_VALUE * 20
+	desc = "Contains two Carwo 'Sindano' Submachineguns, and two spare magazines for them."
+	contains = list(/obj/item/gun/ballistic/automatic/sol_smg = 2,
+	/obj/item/ammo_box/magazine/c35sol_pistol = 2,
+	)
+
+	crate_name = "Carwo 'Sindano' Submachinegun Crate"
+
+/datum/supply_pack/security/armory/infanterie
+	name = "Carwo 'd'Infanteria' Rifle Crate"
+	desc = "Contains two Carwo 'd'Infanteria' Rifles, and two spare magazines for them."
+	cost = CARGO_CRATE_VALUE * 20
+	contains = list(/obj/item/gun/ballistic/automatic/sol_rifle = 2,
+	/obj/item/ammo_box/magazine/c40sol_rifle = 2,
+	)
+	crate_name = "Carwo 'd'Infanteria' Rifle Crate"
+
+/datum/supply_pack/security/armory/elite
+	name = "Carwo 'd'Elite' Sniper Rifle Crate"
+	desc = "Contains one Carwo 'd'Elite' Sniper Rifle, as well as a spare magazine for it."
+	cost = CARGO_CRATE_VALUE * 40
+	contains = list(/obj/item/gun/ballistic/automatic/sol_rifle/marksman = 1,
+	/obj/item/ammo_box/magazine/c40sol_rifle = 1,
+	)
+	crate_name = "Carwo 'd'Infanteria' Rifle Crate"
+
+/datum/supply_pack/security/ammo
```

Comment:
Does this not have a name or description?

Reply:
This is a modular override to an existing crate probably but let me add one just in case